### PR TITLE
🎨 Minor on invitations extra-credits limits

### PR DESCRIPTION
**Pull request by pcrespov**
## What do these changes do?

Aims to complete missing details in tasks associated to @pcrespov in https://github.com/ITISFoundation/osparc-issues/issues/949:
- Account creation: workflow (Done in  https://github.com/ITISFoundation/osparc-simcore/pull/4862 )
- Invitation: 100$ by default (Done here)

## Related issue/s

- Follows from https://github.com/ITISFoundation/osparc-simcore/pull/4862

## DevOps

### CODE REVIEW

File: services/static-webserver/client/source/class/osparc/po/Invitations.js [hunk 1]
```diff
@@ -101,8 +101,8 @@ qx.Class.define("osparc.po.Invitations", {
 
       const extraCreditsInUsd = new qx.ui.form.Spinner().set({
         minimum: 0,
-        maximum: 1000,
-        value: 0
+        maximum: 199,
+        value: 100
```
Comment (GitHK) [inline services/static-webserver/client/source/class/osparc/po/Invitations.js:105]: I would say that these are the credits that one is given when registering.

Q: Who decides this number, the person issuing the invitation? (then this is OK)

**Review by GitHK — APPROVED** [review]

File: packages/models-library/src/models_library/api_schemas_webserver/product.py [hunk 2]
```diff
@@ -48,10 +48,15 @@ class GetProduct(OutputSchema):
     credits_per_usd: NonNegativeDecimal | None
 
 
+class ExtraCreditsUsdRangeInt(ConstrainedInt):
+    ge = 0
+    lt = 200
```
Comment (sanderegg) [inline packages/models-library/src/models_library/api_schemas_webserver/product.py:53]: Q: why 200? 
was that some decision? I am a bit confused, and wondering if this should be hard-coded at all.

**Review by sanderegg — APPROVED** [review]

File: services/static-webserver/client/source/class/osparc/po/Invitations.js (hunk 1 above)
Comment (pcrespov) [reply services/static-webserver/client/source/class/osparc/po/Invitations.js:105]: NK decided. This is used as default.
PO users have access to invitations

File: packages/models-library/src/models_library/api_schemas_webserver/product.py (hunk 2 above)
Comment (pcrespov) [reply packages/models-library/src/models_library/api_schemas_webserver/product.py:53]: yes. NK

**Review by odeimaiz — APPROVED** [review]

**Pull request merged**